src/networks.test.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Network,
+  networkLabel,
+  networkData,
+  getNetworkFromPrefix,
+} from "./networks";
+
+import { networks } from "bitcoinjs-lib";
+
+describe("networks", () => {
+  describe("networkLabel", () => {
+    it("returns a human-readable network name", () => {
+      expect(networkLabel(Network.MAINNET)).toBe("Mainnet");
+      expect(networkLabel(Network.TESTNET)).toBe("Testnet");
+      expect(networkLabel("foobar")).toBe("Testnet");
```

Comment:
lol wat

Reply:
This existed as previous functionality. It was just moved over from another file. It's testing the friendly naming for different networks and that it defaults to "Testnet".

Reply:
Yeah I know. I'm just kind of laughing that this is/was supported to begin with.